subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BuildInitPluginIntegrationTest.groovy

```diff
@@ -389,6 +389,17 @@ class BuildInitPluginIntegrationTest extends AbstractInitIntegrationSpec {
         targetDir.list().size() == 0 // ensure nothing generated
     }
 
+    def "warns when initializing in a project directory that is not empty."() {
```

Comment:
Why you decided to put the dot at the and of test name?
As i see no one of other tests names is not ending with dot. So let's keep this name in consistent with others?

Reply:
Makes sense, fixed.